src/components/Inngest.ts

```diff
@@ -254,7 +262,18 @@ export class Inngest<Events extends Record<string, EventPayload>> {
       );
     }
 
-    const response = await fetch(this.inngestApiUrl.href, {
+    // When sending events, check if the dev server is available.  If so, use the
+    // dev server.
+    let url = this.inngestApiUrl.href;
+
+    if (!isProd()) {
```

Comment:
An interesting point to note here is that we have two different checks for prod - this one and the one within handlers, such as `"inngest/next"`.

It makes sense as we can't reliably detect what environment we're in here, whereas handlers are explicitly imported, so we can assume the user has got it right.

Don't think there's an action here, but raising in case we have a bright idea to reduce any drift.

Reply:
Yea, this is a good idea.  We definitely want to stop prod receiving it in that case.  I'll make that change!